nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -39,6 +39,7 @@ nifi.nar.library.directory=${nifi.nar.library.directory}
 nifi.nar.library.autoload.directory=${nifi.nar.library.autoload.directory}
 nifi.nar.working.directory=${nifi.nar.working.directory}
 nifi.documentation.working.directory=${nifi.documentation.working.directory}
+nifi.nar.unpack.uber.jar=${nifi.nar.unpack.uber.jar}
```

Comment:
Shouldn't this (also) be added to stateless.properties?

Reply:
Not necessarily. With nifi.properties, we tend to offer a lot more properties because it's easy to configure them and because it's a very mature "product." With stateless, this will be important for many cases, and it's going to be hard to expose this in a way that makes sense to most users. I expect this property will go away from NiFi in the future and we'll always do this, but wanted to make it configurable just in case there are any problems. WIth Stateless we take a few more liberties than with NiFi just because it's newer and faster evolving.